source/validations.ts

```diff
@@ -34,6 +35,20 @@ class ValidationError extends Error {
  * The validations that can be run, as well as the methods to run them.
  */
 export class Validations {
+	/**
+	 * Map of request -> store -> keys
+	 *
+	 * Store is either:
+	 *  - an instance for stores like the MemoryStore where two instances do not share state
+	 *  - a string (usually class name) for stores where multiple instances typically share state, such as the redis store
+	 *
+	 * todo: add a reset method (for tests)
```

Comment:
Actually, I don't think I need this, because WeakMaps are awesome. The test ends, the `req` object goes out of scope, and everything gets garbage collected, returning the weakmap to a pristine state.

Reply:
Brilliant 🎉